### DISCUSSION TITLE
Simplified Full Syntax Diagram Syntax with where

### DISCUSSION THREAD

**Discussion by mikemdr1** [duckdb/duckdb#8551] [Ideas]
First of all, to have this duckdb syntax is awesome, it's really nice
However, I noticed there is no a change to filter rows ( in a easy way )
I mean, you can filter the pivot values with IN clause; but maybe I need to filter some category / country / product / etc.. 

The only way to do this is to create a sub-query after PIVOT clause, but not directly in the pivot syntax
I was thinking something like that
![Simplified Full Syntax Diagram Modified](https://github.com/duckdb/duckdb/assets/36677796/d3f57d9d-2b10-449c-9e93-50aa5f63cb73)

In this way, maybe is not so neccesary to have IN clause ( because you can filter pivot values before in WHERE clause, but with more flexibility to filter other columns)

Thanks in advance to read this post

**Comment by l1t1** [comment]
yes
